mesonbuild/backend/ninjabackend.py

```diff
@@ -1492,12 +1492,12 @@ def generate_pch_rule_for(self, langname, compiler, qstr, is_cross, outfile):
             output = ''
         else:
             output = ' '.join(compiler.get_output_args('$out'))
-        command = " command = %s %s $ARGS %s %s %s $in\n" % \
-            (' '.join(compiler.get_exelist()),\
-             ' '.join(cross_args),\
-             ' '.join(quoted_depargs),\
-             output,\
-             ' '.join(compiler.get_compile_only_args()))
+        command = (" command = %s %s $ARGS %s %s %s $in\n" %
+            (' '.join(compiler.get_exelist()),
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->